WordPress/WordPressUITests/Tests/SignupTests.swift

```diff
@@ -26,4 +26,15 @@ class SignupTests: XCTestCase {
 
         XCTAssert(mySiteScreen.isLoaded())
     }
+
+    // Test Support Section Loads
+    // From Prologue > continue, tap "help" and make sure Support Screen loads
+    func testSupportScreenLoads() {
+        let supportScreen = PrologueScreen().selectContinue().selectHelp()
+
+        XCTAssert(supportScreen.isLoaded())
+
+        //close SupportScreen modal before teartown
+        SupportScreen().closeButton.tap()
+    }
```

Comment:
In the PR description you ask:

> One structure question I still have: is SignupTests a good place for this test?
> I put it there because:
> 
> - LoginTests is getting pretty busy
> - It doesn't require logging in (it tests Help from a non-logged-in state)
>   But I'm not sure either reason justifies putting it in SignupTests. Should it have it's own file or be elsewhere?

I like the idea of having it in its own file, because this is sort of part of both the logic and signup flow, but also part of neither. I'm not sure how I would call the file itself, but it doesn't matter _too_ much. Maybe `SupportScreenTests` or `UnauthenticatedSupportScreenTests`? 🤷‍♂️

Reply:
That makes sense to me, and leave room for creating further UI tests of various support screens if desired. I think `SupportScreenTests` makes most sense, so I'll go with that. :)